src/pynwb/base.py

```diff
@@ -287,6 +287,13 @@ def __get_links(self, links):
     def __add_link(self, links_key, link):
         self.fields.setdefault(links_key, list()).append(link)
 
+    def _generate_field_html(self, key, value, level, access_code):
+        # reassign value if linked timestamp or linked data to avoid recursion error
+        if key in ['timestamp_link', 'data_link']:
+            value = {v.name: v.neurodata_type for v in value}
```

Comment:
Rather than link to the linked object, can we link to just the timestamps (or data) of the corresponding object?

Reply:
Do you mean instead of displaying the neurodata_type, show just the timestamps/data of the corresponding object? Or do you mean changing how the timestamp_link and data_link properties get set up?

Reply:
The former. In practice, we do not show arrays yet in this html repor, so nothing will show up, but it would be less confusing as to why timestamps = another NWB object.

It would however be nice to say that this is a linked object, so maybe we could alter the key to say something like 
`→ timestamps (link to <path to timestamps of other NWB object -- use container.get_ancestors() and make a nice string path out of it>)"`

Reply:
Got it, I was just looking at the items in the 'timestamp_link' subheading and didn't realize the timestamps for the other timeseries were still showing the linked object. Yes, I can try to change that as well as alter the key that indicates it is a linked object.

Reply:
Ah right. Yes I was looking at the timestamps field and not the timestamps_link field.